### PR TITLE
Remove usage of strings for queries in test cases

### PR DESCRIPTION
**Pull request by niveathika**
## Purpose
$subject as part of https://github.com/ballerina-platform/ballerina-standard-library/issues/1880

## Checklist
- [x] Linked to an issue
- [ ] ~Updated the changelog~
- [ ] ~Added tests~

### CODE REVIEW

File: ballerina/tests/execute-basic-test.bal [hunk 1]
```diff
@@ -334,6 +329,7 @@ type ResultCount record {
 }
 function testUpdateData() returns error? {
     MockClient dbClient = check new (url = executeDb, user = user, password = password);
+    // Usage of string in the execute API is depreciated
     ExecutionResult result = check dbClient->execute("Update NumericTypes set int_type = 11 where int_type = 10");
```
Comment (daneshk) [inline ballerina/tests/execute-basic-test.bal:333]: why we do not change to parameterized query here?

Comment (daneshk) [reply ballerina/tests/execute-basic-test.bal:333]: we might remove the string option from all APIs soon. So better we move to parameterized queries

Comment (niveathika) [reply ballerina/tests/execute-basic-test.bal:333]: I have left one usage of each API as it is to test the string usage for now. All the others were changed.

Comment (daneshk) [reply ballerina/tests/execute-basic-test.bal:333]: Okay, let's remove them when we remove the support completely

**Review by daneshk — APPROVED** [review]

**Pull request merged**